src/command-bases/live-base.ts

```diff
@@ -0,0 +1,22 @@
+import { flags } from '@oclif/command';
+
+import CommandBase from './base';
+
+export default abstract class LiveCommandBase extends CommandBase {
+  static flags = {
+    streamId: flags.string({
+      name: 'stream-id',
+      char: 'i',
+      description: 'The live stream, by its stream id',
+      required: true,
```

Comment:
I understand why this is happening, but it feels like an antipattern to use a flag instead of an arg. Should we instead make the flag what kind of ID it is? I.e.

```
mux livecmd:option --stream-key abcd123
```

Where stream key is a boolean and the ID is an argument instead. The default would then be a stream id, but still gives us the flexibility to swap that out later.

Reply:
done, `-t stream-id` is now implied